akka-stream/src/main/scala/akka/stream/remote/StreamRefs.scala

```diff
@@ -18,6 +18,14 @@ private[akka] object StreamRefs {
     if (payload == null) throw ReactiveStreamsCompliance.elementMustNotBeNullException
   }
 
+  /** Sent to a the receiver side of a SinkRef, once the sending side of the SinkRef gets signalled a Failure. */
+  @InternalApi
+  final case class RemoteSinkFailure[T](msg: String) extends StreamRefs.Protocol
```

Comment:
Open question if we want to send over the cause.getMessage or not at all. cc @jrudolph @patriknw